src/main/drivers/serial.h

```diff
@@ -44,9 +44,20 @@ typedef enum {
      * to actual data bytes.
      */
     SERIAL_BIDIR_OD      = 0 << 4,
-    SERIAL_BIDIR_PP      = 1 << 4
+    SERIAL_BIDIR_PP      = 1 << 4,
+
+    SERIAL_PARITY        = (1 << 6) | (1 << 5),
+    SERIAL_PARITY_NO     = (0 << 6) | (0 << 5),
```

Comment:
Probably better `SERIAL_PARITY_NONE`, which seems to be the standard term for no parity.

Reply:
Sure, just wanted to keep it aligned with old code :-)

Reply:
Agreed, `SERIAL_PARITY_NONE` is better.